mllib/src/test/scala/org/apache/spark/mllib/tree/GradientBoostedTreesSuite.scala

```diff
@@ -111,8 +115,10 @@ class GradientBoostedTreesSuite extends SparkFunSuite with MLlibTestSparkContext
           EnsembleTestHelper.validateClassifier(gbt, GradientBoostedTreesSuite.data, 0.9)
         } catch {
           case e: java.lang.AssertionError =>
+            // scalastyle:off println
             println(s"FAILED for numIterations=$numIterations, learningRate=$learningRate," +
```

Comment:
What do you mean by "print"? Test logs go to a file (e.g. `core/target/unit-tests.log`), not to the terminal.